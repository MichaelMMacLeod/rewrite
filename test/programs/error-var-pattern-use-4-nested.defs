(def (one ($x two ($x (three $x))) four $x)
  constructor)