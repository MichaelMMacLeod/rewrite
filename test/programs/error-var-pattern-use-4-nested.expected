error[E007]: variable occurs more than once in pattern
./test/programs/error-var-pattern-use-4-nested.defs:1:12
   |
 1 | (def (one ($x two ($x (three $x))) four $x)
   |            ^^ use #1
./test/programs/error-var-pattern-use-4-nested.defs:1:20
   |
 1 | (def (one ($x two ($x (three $x))) four $x)
   |                    ^^ use #2
./test/programs/error-var-pattern-use-4-nested.defs:1:30
   |
 1 | (def (one ($x two ($x (three $x))) four $x)
   |                              ^^ use #3
./test/programs/error-var-pattern-use-4-nested.defs:1:41
   |
 1 | (def (one ($x two ($x (three $x))) four $x)
   |                                         ^^ use #4
help: a variable may occur at most once in a definition's pattern
